Sources/MapboxNavigation/ViewportDataSourceType.swift

```diff
@@ -0,0 +1,28 @@
+/**
+ Possible types of location related updates `NavigationViewportDataSource` can track.
+ `ViewportDataSourceType` can also be used in custom implementations of classes, which conform to
+ `ViewportDataSource`.
+ */
+public enum ViewportDataSourceType {
+    
+    /**
+     Type, which is used to track raw location updates. In case if such type is used
```

Comment:
nit: something like: "If 'raw' type is specified, `NavigationViewportDataSource` will..." sounds better to my ear.

Reply:
Also, "continiously" should be "continuously"

Reply:
Thanks, updated comments.